application/healthcare-services-data-manager/requirements.txt

```diff
@@ -1,4 +1,6 @@
 Flask==1.1.4
-pytest==5.4.1
+# pytest==6.2.5
```

Comment:
Remove dead code

Reply:
How are we making sure that each service uses the same versions of imports?